main.py

```diff
@@ -1,3 +1,11 @@
 print(123)
 print(123456)
+print(123456)
+print(123456)
+print(123456)
+print(123456)
```

Comment:
dont do it like this any more!!!!!

Reply:
e3riopuer3fwoiu;erwfioy;puerwfgrefwoihyerfwijgerwfliugwef

ewd
edw
ewd

edwedw
edw
ewd
ewd
ewd